### PR TITLE
Issue 7 Completed User can sign-up

### PR DESCRIPTION
**Pull request by YoonHCho**
Issue 7 Completed User can sign-up

Final Proj issue 7

### CODE REVIEW

File: client/pages/home.jsx [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
+// export default function Home(props) {
 
-  return (
-    <MapComponent />
-  );
-}
+//   return (
+//     // <MapComponent />
+//   );
+// }
```
Comment (thebearingedge) [inline client/pages/home.jsx]: I think you can delete this file

File: client/lib/parse-route.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export default function parseRoute(hashRoute) {
+  // console.log('hashRoute: ', hashRoute);
```
Comment (thebearingedge) [inline client/lib/parse-route.js]: Please look for commented code

File: client/app.jsx [hunk 3]
```diff
@@ -1,8 +1,10 @@
 import React from 'react';
-import Home from './pages/home';
+// import Home from './pages/home';
+import MapComponent from './components/map-component';
 
 export default class App extends React.Component {
   render() {
-    return <Home />;
+    // return <Home />;
```
Comment (thebearingedge) [inline client/app.jsx]: Please look for commented code

File: server/public/styles.css [hunk 4]
```diff
@@ -253,6 +272,85 @@ img {
...
+
+.sign-up-header {
+  font-family: Inter, sans-serif;
+  margin-bottom: 0;
+}
+
+.sign-up-para {
+  font-family: Inter, sans-serif;
+  font-size: x-small;
+  margin-bottom: 0;
```
Comment (thebearingedge) [inline server/public/styles.css]: There is a _lot_ of duplication in your CSS. try to use some element selectors where it makes sense, like all inputs for fonts.

And for other elements, let them inherit the font. Box sizing border-box should be in your reset.css for * universal selector. Please try to combine a lot of your CSS

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Comment by YoonHCho** [comment]
removed box-sizing, and removed some of the duplicate.
i kept some because i have a media query and it was messing up other modals

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**